scripts/upload_xgboost_mnist_dataset/README.md

```diff
@@ -0,0 +1,15 @@
+# `upload_mnist`
+This script uploads the MNIST dataset that is compatible with the [Amazon SageMaker XGBoost algorithm](https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html) to the specified S3 bucket.
+It downloads the MNIST dataset, splits it into train, test, and validation partitions, then uploads the partitions as CSV files into S3.
```

Comment:
Could we attribute the data set to its source in this README? This would allow us to update in the future, or reference our methodology.